### PR TITLE
Highlight earthaccess in Welcome

### LINKED ISSUE #332: Add earthaccess more prominently to Cookbook homepage
Add earthaccess more prominently to Cookbook, maybe under [Welcome header](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/#welcome), it could be simple like on https://nasa-openscapes.github.io/ homepage, maybe saying 

```
"earthaccess is a python library to access NASA earthdata locally or in the cloud. Learn more here, see this tech spotlight (blog link), this earthdata blogpost, and these tutorials"

Looking for R and MATLAB libaries? see earthdatalogin's page and tutorials (in cookbook), and MATLAB's tutorials (link in cookbook)
```

 What do you all Mentors think? There is a blog coming out about earthdatalogin and this will be great to have!

### PR DESCRIPTION
**Pull request by ateucher**
Closes #332

### CODE REVIEW

**Review by ateucher — COMMENTED** [review]
@stefaniebutland a draft we can discuss with mentors - if you don't mind doing a quick review I will merge and we can discuss at coworking

File: index.qmd [hunk 1]
```diff
@@ -13,6 +13,10 @@ slug: index
 
 This Cookbook is learning-oriented to support scientific researchers who use NASA Earthdata as NASA migrates data and workflows to the cloud. It has resources to learn how to work with NASA Earthdata in the cloud, as well as documentation for how to contribute to these resources. It is under active, open development.
 
+**Tip: use the _earthaccess_ python library to access NASA earth data**. [earthaccess](https://earthaccess.readthedocs.io/en/latest/) is a Python library to access NASA earthdata locally or in the cloud. To learn more, see this [tech spotlight](), the [NASA Earthdata blog post](https://www.earthdata.nasa.gov/learn/blog/earthaccess), and these [how-to's](how-tos/find-data/find-python.html) and [tutorials](tutorials/).
```
Comment (stefaniebutland) [inline index.qmd:16]:
```suggestion
**Tip: use the _earthaccess_ Python library to access NASA Earthdata**. [earthaccess](https://earthaccess.readthedocs.io/en/latest/) is a Python library that streamlines access to NASA Earthdata locally or in the cloud. To learn more, see <!--- this [tech spotlight](),---> the [NASA Earthdata blog post](https://www.earthdata.nasa.gov/learn/blog/earthaccess), and these [how-to's](how-tos/find-data/find-python.html) and [tutorials](tutorials/). 
```

Comment (stefaniebutland) [reply index.qmd:16]: @ateucher I think it's best to merge this now, commenting out the tech spotlight till published, since coworking not till Thursday.

**Review by stefaniebutland — APPROVED** [review]

File: index.qmd (hunk 1 above)
Comment (jules32) [reply index.qmd:16]: @stefaniebutland link to tech spotlight: https://nasa-openscapes.github.io/news/2024-03-04-earthaccess-tech-spotlight/

Comment (stefaniebutland) [reply index.qmd:16]: ah I get it! Thanks @jules32. I updated my suggested edit above. Ready to merge @ateucher

**Pull request merged**